en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md

```diff
@@ -91,3 +91,238 @@ In the following diagram, the five components are set up in a distributed deploy
 [![Understanding the distributed deployment]({{base_path}}/assets/img/setup-and-install/db-connections-distributed-deployment.png)]({{base_path}}/assets/img/setup-and-install/db-connections-distributed-deployment.png)
 
 
+??? "Click here for more details on default communication ports among components."
+    [![Communication ports among components]({{base_path}}/assets/img/setup-and-install/communication-among-port.png)]({{base_path}}/assets/img/setup-and-install/communication-among-port.png)
+    **Open port among profiles**
+    <table>
+    <thead>
+    <tr class="header">
+    <th>Client</th>
+    <th>Server</th>
+    <th>Port</th>
+    <th>Description</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr class="odd">
+    <td>Gateway</td>
+    <td>Key Manager</td>
+    <td>9443</td>
+        <td>
+            Token validation, token, revoke, authorize endpoint, block condition, revoke tokens
+        </td>
+    </tr>
+    <tr class="even">
+    <td>Gateway</td>
+    <td>Analytics Worker</td>
+    <td>7612</td>
+        <td>
+            Publishing analytics events
+        </td>
+     </tr>
+    <tr class="odd">
+    <td>Gateway</td>
+    <td>Analytics Worker</td>
+    <td>7712</td>
+        <td>
+            authenticate to Publishing analytics events
+        </td>
+    </tr>
+    <tr class="even">
+    <td>Gateway</td>
+    <td>Traffic Manager</td>
+    <td>9611</td>
+        <td>
+            Publishing throttling events
+        </td>
+    </tr>
+    <tr class="odd">
+    <td>Gateway</td>
+    <td>Traffic Managerr</td>
+    <td>9711</td>
+        <td>
+            authenticate to Publisherlishing throttling events
+        </td>
+    </tr>
+    <tr class="even">
+    <td>Publisher</td>
+    <td>Traffic Manager</td>
+    <td>9443</td>
+        <td>
+            Deploying throttling policy via admin portal
+        </td>
+    </tr>
+    <tr class="odd">
+    <td>Publisher</td>
+    <td>Traffic Manager</td>
+    <td>9611</td>
+        <td>
+            Publishing events like block condition, token revocation
+        </td>
+    </tr>
+    <tr class="even">
+    <td>Publisher</td>
+    <td>Traffic Manager</td>
+    <td>9711</td>
+        <td>
+            authenticate to Publishing events
```

Comment:
Start with capital. Please fix this in other places as well.

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/2794/commits/8c01a94ac09eb016f84e0d7213401826fe556d83